src/parser/RdfParser.cpp

```diff
@@ -545,6 +545,15 @@ TripleComponent TurtleParser<T>::literalAndDatatypeToTripleComponentImpl(
                   "instead"
                << std::endl;
     parser->lastParseResult_ = std::move(literal);
+  } catch (const CoordinateOutOfRangeException& ex) {
+    LOG(DEBUG) << normalizedLiteralContent
+               << " could not be parsed as GeoPoint object for the following "
+                  "reason: "
+               << ex.what()
+               << ". It is treated as a plain string literal without datatype "
+                  "instead"
+               << std::endl;
+    parser->lastParseResult_ = std::move(literal);
```

Comment:
The handling of these various exceptions in now very redundant.
Can you factor the common code out into a lambda, s.t. we can simply write something like

```
} catch( const CoordinateException& ex) {
  makeNormalLiteral(absl::StrCat(normalizedLiteralContent, "could not be parsed as a GeoPoint object for the following reason: ", ex.what());
}
```

And use the same helper lambda for all the other catch clauses above.
I also thought about factoring out the part "could not be parsed as" and "for the following reason", but that is up to you (some of them use the exception message, others don''t, so i am not sure, if two cases are worth it here, but maybe the 'could not be parsed as' can be generalized.
}